### PR TITLE
Howto for deploying images to OpenStack.

### PR DESCRIPTION
**Pull request by jbzdak**

### CODE REVIEW

**Comment by antoviaque** [comment]
@jbzdak What is the name I had asked to give to this repository?

**Comment by jbzdak** [comment]
Fixed. Sorry I missed that one.

On Mon, May 2, 2016 at 1:29 PM, Xavier Antoviaque notifications@github.com
wrote:

> @jbzdak https://github.com/jbzdak What is the name I had asked to give
> to this repository?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/open-craft/public-doc/pull/1#issuecomment-216209527

## 

Jacek
@OpenCraft

**Comment by antoviaque** [comment]
@jbzdak Thanks! :+1:

File: howto-upload-images-to-openstack.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+   with following short ID: `7DB87C81`
+3. Download signing key: `gpg --keyserver keyserver.ubuntu.com --recv-keys 7DB87C81`
+ 
+Download the images
+-------------------
+ 
+1. Download the images from: https://cloud-images.ubuntu.com/. Example file to download: 
+   https://cloud-images.ubuntu.com/trusty/current/trusty-server-cloudimg-amd64-disk1.img
+2. Download `SHA256SUMS.txt` and `SHA256SUMS.gpg`, verify these files `gpg --verify SHA256SUMS.gpg SHA256SUMS.txt`
+3. Verify downloaded image: `sha256sum --check SHA256SUMS.txt`
```
Comment (antoviaque) [inline howto-upload-images-to-openstack.md:19]: @jbzdak Sorry, I had missed that one - there is no `txt` extension on the `SHA256SUMS` file

File: howto-upload-images-to-openstack.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Upload the images
+-----------------
+
+1. Download the rc file: go to horizon console `Access ans Security` -> `API Access` -> `Download RC File`.
```
Comment (antoviaque) [inline howto-upload-images-to-openstack.md:24]: @jbzdak `s/ans/and/`

**Pull request merged**